docs/_static/custom.css

```diff
@@ -0,0 +1,3 @@
+#version_switcher a[data-version-name*="stable"] {
+  background-color: #e2ffe2;
+}
```

Comment:
Can I suggest putting all the old versions with a light red? Or is that too much? :wink:

Reply:
done!